doc/elements.md

```diff
@@ -105,6 +105,29 @@ def file(data):
 
 If `file()` does not return anything, it will be treated as if `file()` returned the empty string.
 
+## `pl_file_upload` element
+
+```html
+<pl_file_upload files="foo.py, bar.c, filename with\, comma]" />
+```
+
+Provides a way to accept file uploads as part of an answer. They will be stored
+in [the format expected by externally graded questions](externalGrading.md#file-submission-format).
+
+Attribute | Type | Default | description
```

Comment:
The table is correct, the example is not. I'll update that, and I think I'll also switch to `file_names` for consistency with the download element.